content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,61 @@
+
+7. The wallet detects that the account has been created and a trustline must be established.
+8. The wallet prompts the user to add the asset/create the trustline.
+9. Finally, the anchor sends the deposit funds to the user’s Stellar account.
+
+![anchor creates account flow](../../web/assets/first-deposit-anchor-flow.png)
+
+With the flow described above, the wallet and the anchor have to facilitate listening for and responding to the trustline status, which can create user experience frictions when waiting for the trustline to be established. To address this issue, Protocol 15 introduces [Claimable Balances](../glossary/claimable-balance.mdx), which enhance the flow by allowing users to start using the wallet without having to secure XLM wait to create the trustline after they made their first deposit. Both the wallet and the anchor have to implement Claimable Balance support in order to make this flow work.
+
+The flow with Claimable Balances looks like this:
+
+1. The wallet registers a new user, and generates a keypair.
+2. The wallet initiates a deposit on behalf of a user.
+3. The anchor provides deposit instructions to the wallet. 
+4. The user transfers money from a bank account to the anchor’s account.
+5. The anchor creates and funds the user's Stellar account plus the amount required for trustlines and transaction fees.  Again, we suggest 2 XLM to start. 
+6. The anchor creates a Claimable Balance.
+7. The wallet detects the Claimable Balance for the account, claims the funds, and posts it in the wallet. 
+
+![anchor creates account claimable balance flow](../../web/assets/first-deposit-claimable-balance-flow.png)
+
+### Option 2: The wallet creates and funds the Stellar account upon user sign-up
+
+For this option, the wallet creates and funds the Stellar account upon every new user sign-up with the minimum requirement of 1XLM, plus the .5XLM reserve for establishing the first trustline, plus a bit more to cover transaction fees. For more information on minimum balances, check out the [glossary entry](../glossary/minimum-balance.mdx).
+
+The flow looks like this: 
+
+1. Upon a new user signup, the wallet issues a keypair, then creates and funds the user's Stellar account with 2XLM. 
+2. Then the wallet creates a trustline, and initiates the first deposit.
+3. Once the deposit request is sent to the anchor, the anchor provides instructions for the deposit. 
+4. The customer transfer funds from a personal bank account to the anchor’s account.
+5. The anchor receives the funds, then sends them to the user’s Stellar account. 
+6. The wallet detects that funds were sent and notifies the user. 
+
+![wallet creates account flow](../../web/assets/first-deposit-wallet-flow.png)
```

Comment:
These images aren't resolving. I'm not exactly sure, but I think you want `../web-assets/first-deposit-wallet-flow.png`, etc.

Reply:
It's fixed.